initdb.py

```diff
@@ -0,0 +1,140 @@
+# Initializes a MySQL Database
+# 1) connect to server
+# 2) create database with cursor
+# 3) connect to database
+# 4) create tables with cursor
+# 5) commit transactoins
+# 6) exit
+
+
+
+
+import sys
+import mysql.connector
+from mysql.connector import errorcode
+
+# Define constants
+DB_NAME = "starbucksdb"
+
+TABLES = {}
+# Create table statments
+# replace with own schema
+
+# https://www.kaggle.com/starbucks/store-locations
+TABLES['locations']=(
+    "CREATE TABLE `locations` ("
+    "   `brand` varchar(15) NOT NULL,"
+    "   `store_number` varchar(20) NOT NULL,"
+    "   `store_name` varchar(15) NOT NULL,"
+    "   `ownership_type` varchar(10) NOT NULL,"
+    "   `street_address` varchar(25) NOT NULL,"
+    "   `city` varchar(15) NOT NULL,"
+    "   `state` varchar(2) NOT NULL,"
+    "   `country` varchar(2) NOT NULL,"
+    "   `postcode` varchar(10),"
+    "   `phone_number` varchar(11),"
+    "   `timezone` varchar(20) NOT NULL,"
+    "   `longitude` decimal(4,2) NOT NULL,"
+    "   `latitude` decimal(4,2) NOT NULL"
+    ") ENGINE=InnoDB")
+
+# https://github.com/kdallas2/diversity/blob/master/di.csv
+TABLES['diversity'] = (
+    "CREATE TABLE `diversity` ("
+    "   `location` varchar(15) NOT NULL,"
+    "   `diversity_index` decimal(7,6) NOT NULL,"
+    "   `african_american` decimal(4,2) NOT NULL,"
+    "   `american_indian` decimal(4,2) NOT NULL,"
+    "   `asian` decimal(4,2) NOT NULL,"
+    "   `pacific_islander` decimal(4,2) NOT NULL,"
+    "   `two_or_more` decimal(4,2) NOT NULL,"
+    "   `hispanic_latino` decimal(4,2) NOT NULL,"
+    "   `white_only` decimal(4,2) NOT NULL"
+    ") ENGINE=InnoDB")
+
+
```

Comment:
So, I actually have to disagree here. Functions outside of classes should have two empty lines between them and the thing above\